### PR TITLE
[INT-185] Use a random UUID generated server side as the csrf token

### PR DESCRIPTION
**Pull request by archessmn**

### CODE REVIEW

File: app/login/page.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import Image from "next/image";
 import BG from "./login-bg.png";
-import { GoogleLoginButton } from "./GoogleLoginButton";
+import { GoogleLoginButton } from "../../components/google/GoogleLoginButton";
```
Comment (Keeeeeeeen) [inline app/login/page.tsx:3]: Should this not be @/components/... ?

Comment (archessmn) [reply app/login/page.tsx:3]: Good spot, that was vscode doing a funny

Comment (Keeeeeeeen) [reply app/login/page.tsx]: Figured as much, assume this was from a "do you want to update the import statements" when you moved it? aha

Comment (archessmn) [reply app/login/page.tsx]: You guessed it :joy:

**Pull request merged**